Identifier.spec.ts

```diff
@@ -89,4 +89,21 @@ describe("Identifier", () => {
 		expect(cryptly.Identifier.previous("zzz-")).toEqual("zzyz")
 		expect(cryptly.Identifier.previous("----")).toEqual("zzzz")
 	})
+	it.each([4, 8, 12, 16, 64] as const)(`ordered id is`, (length: cryptly.Identifier.Length) => {
+		expect(cryptly.Identifier.is(cryptly.Identifier.generate(length, "ordered", length), length)).toBeTruthy()
+	})
+	const now = new Date().getTime()
+	it.each([4, 8, 12, 16, 64] as const)(`ordered id is ordered`, (length: cryptly.Identifier.Length) => {
+		expect(
+			cryptly.Identifier.generate(length, "ordered", now + 1) > cryptly.Identifier.generate(length, "ordered", now)
+		).toBeTruthy()
+	})
+	it.each([4, 8, 12, 16, 64] as const)(`reverse ordered id is`, (length: cryptly.Identifier.Length) => {
+		expect(cryptly.Identifier.is(cryptly.Identifier.generate(length, "reversed", length), length)).toBeTruthy()
```

Comment:
maybe double check the length of the result.

Reply:
and if i understand the value correctly. then you should be able to regex match the start of the string in these tests? use `expect().toMatch(/.../)` for more accurate tests.

Reply:
If you use the same "seed" on all the tests you can use `startsWith(x)` on the Identifier with x being a precalculated value.